recipes/benchmark/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "benchmark" %}
+{% set version = "1.4.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/google/benchmark/archive/v{{ version }}.tar.gz
+  sha256: f8e525db3c42efc9c7f3bc5176a8fa893a9a9920bbd08cef30fb56a51854d60d
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - test -f $PREFIX/include/benchmark/benchmark.h  # [unix]
+    - if not exist %LIBRARY_INC%\\benchmark\\benchmark.h exit 1  # [win]
+    - test -f $PREFIX/lib/libbenchmark.a  # [unix]
+    - test -f $PREFIX/lib/libbenchmark_main.a  # [unix]
+    - if not exist %PREFIX%\\Library\\lib\\benchmark.lib exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\benchmark_main.lib exit 1  # [win]
+
+about:
+  home: https://github.com/google/benchmark
+  license: Apache 2.0
```

The Apache license requires that the license text is always distributed with the package. Can you add the `license_file` field.

```suggestion
  license: Apache 2.0
  license_family: Apache
  license_file: 'LICENSE'
```